components/servo/lib.rs

```diff
@@ -762,6 +776,31 @@ where
                     self.embedder_events.push(event);
                 },
 
+                (EmbedderMsg::MediaSessionEvent(event), ShutdownState::NotShuttingDown) => {
```

Comment:
In the light of the suggestion at https://github.com/servo/servo/pull/24499/files#r336776086, it might make sense to do this in the constellation, and make the `active_media_session` a `BrowsingContextId`(not a top-level). 

It might then make sense to make the `EmbedderMsg::MediaSessionEvent` into a [`ScriptMsg`](https://github.com/servo/servo/blob/882dcc8408159bed5633337d11e83b6839f97e34/components/script_traits/script_msg.rs#L116) instead, where the constellation would run the below logic and then potentially forward the event to the embedder, potentially containing the `TopLevelBrowsingContextId` representing the "tab" where the event came from(while it might have come from an iframe nested inside of it).